Utils/data_analys.py

```diff
@@ -0,0 +1,52 @@
+import csv
+import pandas as pd
+from collections import Counter
+from nltk import ngrams
+from nltk.tokenize import TweetTokenizer
+import spacy
+import re
+
+
+DATA_DIR = '../../Data/Russian Twitter Corpus/'
+
+
+def read_csv_clear(name):
```

Comment:
Let's add a small documentation for each our function :) it should describe a function result and function parameters..